CHANGELOG.md

```diff
@@ -1,12 +1,27 @@
 # Revision history for hiedb
 
+## 0.6.0.0 -- 2024-12-11
```

Comment:
I think this was supposed to be 2024-02-11?

Reply:
Oops, you're right. I'll push a fix